pkgs/by-name/cl/clang-uml/package.nix

```diff
@@ -72,6 +76,12 @@ stdenv.mkDerivation (finalAttrs: {
   dontFixup = debug;
   dontStrip = debug;
 
+  nativeInstallCheckInputs = [ versionCheckHook ];
+  doInstallCheck = true;
+  versionCheckProgramArg = "--version";
+
+  passthru.tests.version = testers.testVersion { package = clang-uml; };
```

Comment:
Double insurance??

Reply:
Both are acceptable for me, but I don't think it will make sense to add both at the same time. It adds meaningless build overhead.